src/plugins/prebuilt-library/data-fetch/get-async-data.js

```diff
@@ -419,6 +419,31 @@ export function getAsyncData() {
 		}
 	}
 
+	/**
+	 * Send Event to Backend.
+	 *
+	 * @param {string} event_label
+	 * @param {object} event_data
+	 *
+	 * @return {Promise<object>} Promise returns object
+	 */
+	async function sendEvent( event_label, event_data ) {
+		try {
+			const response = await apiFetch( {
+				path: '/kb-design-library/v1/handle_event',
+				method: 'POST',
+				data: {
+					event_label: event_label,
+					event_data: event_data ? JSON.stringify(event_data) : '',
```

Comment:
If you update the JSON schema, you wouldn't need to `JSON.stringify` this any longer.